pkg/dashboard/list.js

```diff
@@ -422,7 +442,7 @@ PageDashboard.prototype = {
             self.plots.forEach(function (p) { p.refresh(); });
         }
 
-        function plot_add(addr) {
+        function plot_add(addr, machine_info) {
```

Comment:
I thought you were going to rename the parameter? or did that newly creep in?

Reply:
Yeah, I guess I just forgot to actually do it...